database/entity/0046-messages_tables/Message.ts

```diff
@@ -0,0 +1,33 @@
+import { BaseEntity, Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm'
+import { Contribution } from '../Contribution'
+
+@Entity('messages', { engine: 'InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci' })
+export class Message extends BaseEntity {
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
+  id: number
+
+  @Column({ name: 'contribution_id', unsigned: true, nullable: false })
+  contributionId: number
+
+  @ManyToOne(() => Contribution, (contribution) => contribution.messages)
+  @JoinColumn({ name: 'contribution_id' })
+  contribution: Contribution
+
+  @Column({ name: 'user_id', unsigned: true, nullable: false })
+  userId: number
+
+  @Column({ length: 2000, nullable: false, collation: 'utf8mb4_unicode_ci' })
+  message: string
+
+  @Column({ type: 'datetime', default: () => 'CURRENT_TIMESTAMP', name: 'created_at' })
+  createdAt: Date
+
+  @Column({ type: 'datetime', default: null, nullable: true, name: 'updated_at' })
+  updateAt: Date
+
+  @Column({ type: 'datetime', default: null, nullable: true, name: 'deleted_at' })
+  deletedAt: Date
+
+  @Column({ name: 'deleted_by', default: null, unsigned: true, nullable: true })
+  deletedBy: number
+}
```

Comment:
Wir benötigen noch eine Spalte "type", die im Konzept beschrieben ist, um bei den Messages einer Contribution unterscheiden zu können, ob es sich um eine "DIALOG" Message oder um eine "HISTORY" Message handelt. Diese wird benötigt, um im FE eine unterschiedliche Darstellung zu ermöglichen, vielleicht brauchen wir das aber auch noch im BE, für verschiedene Logiken/Filter o.ä.